taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -26,6 +26,7 @@ const StartProject = ({ clearNavBarLinks }) => {
   const [description, setDescription] = useState("");
   const [loading, setLoading] = useState(false);
   const [errorMessage, setErrorMessage] = useState("");
+  const [showBranchNameError, setShowBranchNameError] = useState(false);
```

Comment:
no need for this to be a state, it could be 
`const validBranchName  = RegexpValidation.test(branchName)`
and used in the `error` and `helperText` props

Reply:
Yes @nobeeakon I will change that! Thanks.